### ISSUE TITLE
contains cookies should default to no in default template

### ISSUE THREAD

**Issue by draeger-charles-wilson** [Microsoft/threat-modeling-templates#15]
Presently in the default template (default.tb7) the "contains cookies" attribute is yes by default. All other attributes default to a neutral / negative value.

**Comment by edmundgr** [comment]
I agree that this is a usability pain point. We have to set Cookies to No on each an every flow.
